minecraft

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Move minecraft profile if detected at old location
+if [ -d $XDG_DATA_HOME/../.minecraft ]; then
+    if [ ! -f $XDG_DATA_HOME/.migrated ]; then
+    	echo "Migrating to XDG dirs"
+	cp -frl $XDG_DATA_HOME/../.minecraft/* $XDG_DATA_HOME/
```

Comment:
whitespace

Reply:
@p1u3o Please, fix the whitespace. Otherwise, LGTM!

Reply:
Sorry, pushed. I guess that commit had been been sitting on my computer for too long.